packages/api-types/src/index.ts

```diff
@@ -204,5 +206,5 @@ export interface Resource {
 }
 
 export interface GetLemmaResourcesResponseBody {
-  data: Resource[];
+  data: Resource[][];
```

Comment:
Rather than being an array of arrays. Can we map from lemma id to resource list?

Reply:
Done